src/schema_tools/coerce.cljc

```diff
@@ -236,12 +235,17 @@
       (catch #?(:clj Exception, :cljs js/Error) _ x))
     x))
 
+(defn keyword->string [x]
+  (cond-> x
+          (keyword? x) name))
```

Comment:
this doesn't retain the namespace in qualified keywords

Reply:
Will fix good catch :+1: